go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/kong/deck v1.27.1
 	github.com/kong/go-kong v0.47.0
 	github.com/kong/kubernetes-telemetry v0.1.1
-	github.com/kong/kubernetes-testing-framework v0.40.0
+	github.com/kong/kubernetes-testing-framework v0.39.2-0.20231011203028-366e9fe9e610
```

Comment:
This is TBD. There are changes that this test framework needs to work: https://github.com/Kong/kubernetes-testing-framework/compare/kic-isolated-integration-tests

For example, not setting the node port to prevent collisions

Reply:
Removed. Using the release version of ktf now.